rust/statement/thing/mod.rs

```diff
@@ -135,24 +135,22 @@ impl fmt::Display for RolePlayer {
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub struct AttributeValueStatement {
     span: Option<Span>,
-    pub type_: Option<TypeRef>,
+    pub var: Variable,
```

Comment:
@dmitrii-ubskii , I'm probably missing something but why was this `Option<TypeRef>` ?